docs/usage.md

```diff
@@ -160,7 +178,30 @@ format.
 
 Example:
 ```
+./qsimh_amplitudes.x -c ../circuits/circuit_q40 -d 47 -k 0,1,2,3,4,5,6,7,8,9,10,13,14,15,16,17,23,24 -t 8 -w 0 -p 0 -r 4 -i ../circuits/bitstrings_q40_s1 -o ampl_q40_s1_w0 -v 1
+```
+Note that this could take a very long time to run, since parallelism on a
```

Comment:
It will take a very long time (a few weeks or so). This is a large circuit with 15 fSim gates on the cut. There are 4^15 paths in total.

Reply:
Made the comment more explicit, and added warnings against running this command directly.